feature/creation/src/main/java/com/android/developers/androidify/creation/CreationViewModel.kt

```diff
@@ -93,6 +96,15 @@ class CreationViewModel @AssistedInject constructor(
         }
     }
 
+    fun launchImageChooser(launcher: ManagedActivityResultLauncher<PickVisualMediaRequest, Uri?>) {
+        try {
+            launcher.launch(PickVisualMediaRequest(PickVisualMedia.ImageOnly))
```

Comment:
Generally we've been trying to avoid starting activities from the ViewModel as the ViewModel can outlive the Fragment/Composable. Is it possible to keep this in the Composable? Also why doesn't the device have a media picker 😄